vnext/src/Libraries/Components/Flyout/FlyoutProps.ts

```diff
@@ -2,10 +2,26 @@
 // Licensed under the MIT License.
 import { ViewProps } from 'react-native';
 
+export enum Placement {
```

Comment:
We're trying to go away from enums and instead define a type that ORs a bunch of constants together.  enums generate unnecessary code size to convert enum values back to strings (though in this case the enum values are strings so maybe thats less of an issue here).  I guess I'd look in the output js for this file we still probably would see size benefit to not having the enum conversion code in generates at the end of the num.

Reply:
But in this case there are so many string constants that the concatenated list had to be split over a couple of lines. I thought it was too ugly to consume. However, I could just convert the placementOptions field into a simple string and just ignore the enum.

Reply:
...also, I pushed another commit that changes the option text to be clearer.

Reply:
Not a string.  In TS you can have a type that is :  'foo' | 'bar'.  Which means that its sort of a string, but the only valid values are the string 'foo' and the string 'bar'

Reply:
I can work on a commit that changes the enum to a string literal type.

Reply:
I pushed another commit that changes the enum to a string literal type.